AdyenCard/Form/FormCardNumberItem.swift

```diff
@@ -105,6 +105,51 @@ internal final class FormCardNumberItem: FormTextItem, AdyenObserver {
         builder.build(with: self)
     }
     
+    @discardableResult
+    internal func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        
+        guard
+            let text = textField.text,
+            let textRange = Range(range, in: text),
+            let selectedTextRange = textField.selectedTextRange
+        else { return true }
+        
+        let editingDirection = string.count - range.length
```

Comment:
It's a bit confusing to see a direction being `Int`

Reply:
Agree - renamed 👌